freetextresponse/mixins/fragment.py

```diff
@@ -82,7 +82,6 @@ def build_fragment(
             else:
                 item = '../public/' + item
                 data = pkg_resources.resource_string(__name__, item)
```

Comment:
Undoing this change because I found it to be [against the docs](https://mail.python.org/pipermail/distutils-sig/2011-March/017495.html).

```suggestion
                data = pkg_resources.resource_string(__name__, item)
                data = data.decode('utf8')
```